include/functions.h

```diff
@@ -110,7 +114,7 @@ void func_80370A48_8441F8(void);
 void func_803713C8_844B78(s32);
 void func_8037172C_844EDC(s32);
 void func_803717E8_844F98(void);
-GObj* func_80371D14_8454C4(void (*gfxFunc)(GObj*), s32 arg1, Sprite* sprite);
+// GObj* func_80371D14_8454C4(void (*gfxFunc)(GObj*), s32 arg1, Sprite* sprite);
```

Comment:
I had to uncomment this too for context reasons, but we should add it back and probably soon fix the header ordering so it's not as broken

Reply:
Sure, I've added it back in.